app/javascript/packs/containers/QuestionnaireRoutes.jsx

```diff
@@ -43,14 +45,14 @@ class QuestionnaireRoutes extends React.Component {
  * Used by Routes.
  */
 function renderWithProps(ComponentName, props) {
-  return () => (
-    <ComponentName {...props} />
+  return (navProps) => (
+    <ComponentName {...navProps} {...props} />
   );
 }
 
 function mapStateToProps(state, ownProps) {
   return {
-    building: state.buildings[ownProps.match.params.bId]
+    building: state.buildings[ownProps.match.params.bId],
```

Comment:
remove trailing comma

Reply:
i thought it was good practice to leave trailing commas in js so we could add more stuff